spec/features/dashboard/work_version_form_spec.rb

```diff
@@ -243,6 +243,122 @@
         end
       end
     end
+
+    context 'when selecting a work type that uses the data and code deposit pathway' do
+      it 'shows only the fields for scholarly works' do
```

Comment:
This should be fields for data and code.

Reply:
Good eye!